index.js

```diff
@@ -233,7 +238,7 @@ NYC.prototype.walkAllFiles = function (dir, visitor) {
 }
 
 NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
-  var instrument = this.exclude.shouldInstrument(filename, relFile)
+  var instrument = (!this.instrument && this.all) || this.exclude.shouldInstrument(filename, relFile)
```

Comment:
I don't think we want this bit of logic `(!this.instrument && this.all)`, we still want to avoid instrumenting files that are hit by our `exclude` rule, otherwise we'll end up instrumenting a huge chunk of the `node_modules` folder, which slows coverage to a crawl.